services/subnet_manager/src/main/java/com/futurewei/alcor/subnet/service/SubnetService.java

```diff
@@ -2,18 +2,15 @@
 
 import com.futurewei.alcor.common.db.CacheException;
 import com.futurewei.alcor.common.exception.*;
-
 import com.futurewei.alcor.subnet.exception.*;
+import com.futurewei.alcor.web.entity.ip.IpAddrRequest;
+import com.futurewei.alcor.web.entity.mac.MacStateJson;
 import com.futurewei.alcor.web.entity.port.PortEntity;
-import com.futurewei.alcor.web.entity.route.InternalRouterInfo;
+import com.futurewei.alcor.web.entity.route.RouteWebJson;
 import com.futurewei.alcor.web.entity.subnet.NewHostRoutes;
 import com.futurewei.alcor.web.entity.subnet.SubnetEntity;
```

Comment:
@songxiaoyan Your IDE seems re-arrange the import statements in the code.

Reply:
Merge code and resolve conflicts, it automatically import, I tried to turn it off.